dataflow/src/main/java/org/checkerframework/dataflow/cfg/node/ClassDeclarationNode.java

```diff
@@ -0,0 +1,58 @@
+package org.checkerframework.dataflow.cfg.node;
+
+import com.sun.source.tree.ClassTree;
+import java.util.Collection;
+import java.util.Collections;
+import org.checkerframework.javacutil.TreeUtils;
+
+/**
+ * A node representing a class declaration that occurs within a method, for example, an anonymous
+ * class declaration. In contrast to a top-level class declaration, such a declaration has an
+ * initialization store that contains captured variables.
+ */
+public class ClassDeclarationNode extends Node {
```

Comment:
@lazaroclapp @cushon Just a heads-up that we added this new CFG node, which is also now used by ObjectCreationNode. We use this to keep a Store for the location where a nested/anonymous class is declared, fixing #266 
All comments welcome.